src/App.js

```diff
@@ -105,142 +119,80 @@ function App() {
             });
     }, []);
 
+    useEffect(() => {
+        if (isZipValid(filters.filterByZipCode.zipCode)) {
+            setSortBy("miles");
+        }
+    }, [filters]);
```

Comment:
This needs an **else** with `setSortBy("name");`

To test, type in a ZIP Code, [ Update List ].
The list should be sorted by ZIP Code.
Change ZIP Code to blank or invalid, [ Update List ].
The list should be sorted by name of the location.